### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by yunfeiguo**
- [ ] [RES-XXXX](https://binatechnologies.atlassian.net/browse/RES-XXXX)
- [ ] Blocking PRs: ADD HERE IF ANY
* People
  - [ ] Reviewers (>=1): ADD HERE
  - [ ] Merger (>=1): ADD HERE
  - [ ] FYI: @roger-liu-bina 
- [ ] Release note (if needed)
- [ ] No code copied from outside Bina 
- [x] Tests (Unit, Workflow, ITL & Gherkin) are passing
- [ ] PR labels, milestones & assignees

# Summary

minor improvements
keep chr names intact

### CODE REVIEW

File: src/main/java/com/bina/varsim/types/variant/Variant.java [hunk 1]
```diff
@@ -611,7 +611,14 @@ public VariantType getType(final int ind) {
...
         returned.
          */
-        Alt alt = alts[ind - 1];
+        Alt alt = null;
+        try {
+            alt = alts[ind - 1];
+        } catch (IndexOutOfBoundsException e) {
+            log.error(this.toString());
+            e.printStackTrace();
+            System.exit(1);
```
Comment (marghoob) [inline src/main/java/com/bina/varsim/types/variant/Variant.java]: We should not exit here. But instead throw the exception again. As for the stack trace, the JVM would report the stack trace when the exception is handled at the highest level, so we can probably skip it.

**Comment by yunfeiguo** [comment]
@marghoob ok to merge?

File: src/main/java/com/bina/varsim/types/variant/Variant.java [hunk 2]
```diff
@@ -611,7 +611,13 @@ public VariantType getType(final int ind) {
         we can be assured that correct variant type is
         returned.
          */
-        Alt alt = alts[ind - 1];
+        Alt alt = null;
+        try {
+            alt = alts[ind - 1];
+        } catch (IndexOutOfBoundsException e) {
+            log.error(this.toString());
+            throw new IndexOutOfBoundsException();
```
Comment (marghoob) [inline src/main/java/com/bina/varsim/types/variant/Variant.java]: just use `throw e;` to rethrow the same exception.

**Comment by marghoob** [comment]
@yunfeiguo where are the commits to keep the `chr` names intact?

**Comment by yunfeiguo** [comment]
just pushed. sorry.

File: src/main/java/com/bina/varsim/types/variant/Variant.java (hunk 2 above)
Comment (yunfeiguo) [reply src/main/java/com/bina/varsim/types/variant/Variant.java]: Thanks. Fixed.

**Comment by yunfeiguo** [comment]
fix for #144

**Comment by marghoob** [comment]
👍  Feel free to merge and refer this pull request in #144

**Comment by yunfeiguo** [comment]
Thanks. Merging.

**Pull request merged**